package.json

```diff
@@ -10,7 +10,7 @@
     "react-scripts": "1.0.17"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "PORT=80 react-scripts start",
```

Comment:
@benweatherman not sure if this is the best place to tell react not to use the default 3000 port. Or if it's better to set the PORT. Nor am I even sure if this is available to change for production.
https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#advanced-configuration 

Something we can screw around with later, unless you have experience here.

Reply:
A `.env` file might be our best bet so we don't need to configure it for each environment. We'd have to make sure we're copying that file into our docker image. It might also be more trouble than it's worth to use the non-default port so maybe 3000 is fine.

Chef's choice 😸